### PR TITLE
Add userManagement.getJwksUrl

### PR DESCRIPTION
**Pull request by cmatheson**
## Description

This is the url to the JWKS that signs access tokens

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: src/user-management/user-management.ts [hunk 1]
```diff
@@ -544,4 +544,8 @@ export class UserManagement {
 
     return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${sessionId}`;
   }
+
+  getJwksUrl({ clientId }: { clientId: string }): string {
```
[the remark this answers is not in the document]
Comment (cmatheson) [reply src/user-management/user-management.ts:548]: I thought we had standardized on always taking an object in our api methods. Will change this to a string.

[the remark this answers is not in the document]
Comment (PaulAsjes) [reply src/user-management/user-management.ts]: 👍 I'd argue that this isn't an API method but instead an SDK helper method.

File: src/user-management/user-management.ts [hunk 2]
```diff
@@ -544,4 +544,8 @@ export class UserManagement {
 
     return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${sessionId}`;
   }
+
+  getJwksUrl(clientId: string): string {
+    return `${this.workos.baseURL}/sso/jwks/${clientId}`;
```
Comment (PaulAsjes) [inline src/user-management/user-management.ts:549]: I think we should add some checks in here to make sure users don't trip up. Let's check that `clientId` isn't an empty string and that it starts with `client_`. Let's also add tests for these edge cases.

**Review by PaulAsjes — CHANGES_REQUESTED** [review]

File: src/user-management/user-management.ts (hunk 2 above)
Comment (cmatheson) [reply src/user-management/user-management.ts:549]: We can't check for `client_` because not all clients start with that (lots start with `project_`, for example (i'm not sure if there are other variations). i'll add a check for empty strings though.

**Review by PaulAsjes — APPROVED** [review]
